geocat_rtd/static/js/gctools.js

```diff
@@ -39,37 +39,76 @@ jQuery.getQueryParameters = function(s) {
 };
 
 /**
- * Small JavaScript module for the documentation.
+ * Small JavaScript module to fix http://localhost:8080/geoserver examples.
+ *
+ * live=instance.geocat.live results in https://instance.geocat.live/mapping
+ * domain=domain.net results in https://domain.net/geoserver
  */
-var Documentation = {
+var FixLocalHostExamples = {
 
     init : function() {
+        console.log("documentation example...");
         this.rewriteLiveUrls();
     },
     rewriteLiveUrls: function() {
 
         var params = $.getQueryParameters();
-        var terms = (params.live) ? params.live[0].split(/\s+/) : [];
-        if (terms.length) {
-
+        
+        var live = (params.live) ? params.live[0].split(/\s+/) : [];
+        var domain = (params.domain) ? params.domain[0].split(/\s+/) : [];
+        var geoserverSearch = "http://localhost:8080/geoserver";
+        var geoserverReplace;
+        var geonetworkSearch = "http://localhost:8080/geonetwork";
+        var geonetworkReplace;
```

Comment:
@MichelGabriel I found this was the only javascript in our theme, hooking it up I was able to get it to work.

Note separate handling for ``?live=instance`` due to the ``geoserver`` --> ``mapping`` and ``geonetwork`` --> ``catalogue`` URL changes.

There is an ``examples.html`` page included in the sample workbook showing the functionality in action.